### PR TITLE
Whats new 12 06 aws logs

### PR DESCRIPTION
**Pull request by jdanna204**
Publish Monday Dec 9.2024
No PMM approval needed

### CODE REVIEW

File: src/content/whats-new/2024/12/whats-new-12-06-AWS-logs.md [hunk 1]
```diff
@@ -0,0 +1,15 @@
+title: 'Forwarding logs from AWS services to New Relic’
+summary: 'New Relic snow supports forwarding logs from AWS services.’
+releaseDate: '2024-12-09'
+learnMoreLink: 'https://deploy-preview-19463--docs-website-netlify.netlify.app/install/aws-logs/?service=aws_services'
```
Comment (Tcohn1) [inline src/content/whats-new/2024/12/whats-new-12-06-AWS-logs.md:4]: learnMoreLink: 'https://docs.newrelic.com/install/aws-logs/?service=aws_services'

File: src/content/whats-new/2024/12/whats-new-12-06-AWS-logs.md [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+getStartedLink: 'https://deploy-preview-19463--docs-website-netlify.netlify.app/docs/logs/get-started/get-started-log-management/’
+---
+# Forwarding logs from AWS services to New Relic
+New Relic now supports forwarding logs from AWS services. Our AI-powered log management solution provides instant insights from contextualization with all telemetry data across your entire stack to optimize operations and business outcomes. Forwarding logs from AWS Services amplifies your visibility into application and infrastructure performance data (events, errors, traces, and more) to reduce mean time to resolution (MTTR) and quickly troubleshoot production incidents. Our AI-powered parsing and recommendations give you unparalleled visibility into your system's performance so you catch issues before they become problems, and make data-driven decisions with ease. Enhance your team's operational efficiency while maintaining security and compliance.
+To get started:
+1. [Read our documentation](https://deploy-preview-19463--docs-website-netlify.netlify.app/install/aws-logs/?service=aws_services)
```
Comment (Tcohn1) [inline src/content/whats-new/2024/12/whats-new-12-06-AWS-logs.md:10]: [read our documentation] (https://docs.newrelic.com/install/aws-logs/?service=aws_services)

File: src/content/whats-new/2024/12/whats-new-12-06-AWS-logs.md [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+summary: 'New Relic snow supports forwarding logs from AWS services.’
+releaseDate: '2024-12-09'
+learnMoreLink: 'https://deploy-preview-19463--docs-website-netlify.netlify.app/install/aws-logs/?service=aws_services'
+getStartedLink: 'https://deploy-preview-19463--docs-website-netlify.netlify.app/docs/logs/get-started/get-started-log-management/’
+---
+# Forwarding logs from AWS services to New Relic
+New Relic now supports forwarding logs from AWS services. Our AI-powered log management solution provides instant insights from contextualization with all telemetry data across your entire stack to optimize operations and business outcomes. Forwarding logs from AWS Services amplifies your visibility into application and infrastructure performance data (events, errors, traces, and more) to reduce mean time to resolution (MTTR) and quickly troubleshoot production incidents. Our AI-powered parsing and recommendations give you unparalleled visibility into your system's performance so you catch issues before they become problems, and make data-driven decisions with ease. Enhance your team's operational efficiency while maintaining security and compliance.
```
Comment (Tcohn1) [inline src/content/whats-new/2024/12/whats-new-12-06-AWS-logs.md:8]: We are thrilled to announce New Relic's latest advancement: one-step observability for AWS, now enhanced with streamlined log forwarding capabilities. Our comprehensive observability solution empowers enterprises by offering seamless integration and enhanced visibility into AWS services within minutes, all within a single user interface.

### Key Benefits of New Relic's Log Forwarding Solution

- **Accelerate Instrumentation:** Set up instrumentation within minutes for forwarding logs and metrics, facilitated by CloudFormation templates and consolidated multi-services onboarding—all accessed through a unified interface.

- **Enhanced Log Management**: Once onboarded, New Relic's Intelligent Observability Platform automatically handles both infrastructure and application performance monitoring, eliminating the need for additional configurations and significantly improving operational efficiency.

- **Improve Operational Efficiency**: Boost your team's operational efficiency while maintaining stringent security and compliance. New Relic's API keys can now be securely stored in AWS Secrets Management, optimizing operations to seamlessly achieve better business outcomes.

- **Reduced Mean Time to Resolution (MTTR)**: New Relic's centralized platform facilitates quick troubleshooting and speeds up incident resolution with a clear view of performance data and relationships across applications, infrastructures, and AWS services out-of-the-box.


###How It Works

1. **Set Up Integration with CloudFormation**: Effortlessly configure AWS integration using the CloudFormation template. Forward metrics and logs by selecting an AWS service and setting up log triggers via S3 or CloudWatch.
2. **Automatic Data Correlation**: Automatically correlate performance data, allowing you to prioritize and address critical issues without the complexity of manual tagging.
3. **Comprehensive Data Querying**: Easily access and analyze log data through the logs UI or New Query Explorer, filtering logs via the Lambda forwarder or Kinesis Data Firehose.

File: src/content/whats-new/2024/12/whats-new-12-06-AWS-logs.md [hunk 4]
```diff
@@ -0,0 +1,15 @@
...
+To get started:
```
Comment (Tcohn1) [inline src/content/whats-new/2024/12/whats-new-12-06-AWS-logs.md:9]: **Get started today** to achieve a unified, holistic view of your AWS environment, enabling data-driven decision-making while ensuring security and compliance. Discover how New Relic's logs management can boost productivity and help you securely scale your cloud-native applications.

**Review by Tcohn1 — CHANGES_REQUESTED** [review]
updated language and links

**Comment by sujitnewrelic** [comment]
Due to build issue, migrated the content to https://github.com/newrelic/docs-website/pull/19487 and merged succesfully. Hence closing this redundant PR.

**Pull request closed without merging**